Add 406 as a valid response code

ruudvanderweijde opened the pull request:
When the response code is 406, the api returned a valid response.

More info on error codes:
https://postmarkapp.com/developer/api/overview#error-codes
> 406 — Inactive recipient You tried to send email to a recipient
> that has been marked as inactive. Inactive recipients have either
> generated a hard bounce or a spam complaint. In this case, only
> hard bounce recipients can be reactivated by searching for them on
> your server’s Activity page and clicking the “Reactivate” button.

ruudvanderweijde committed:
Merge pull request #1 from Werkspot/be_able_to_throw_exceptions

Be able to throw exceptions

ferrastas commented on src/Postmark/Transport.php:
```diff
@@ -95,8 +95,9 @@ public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = nul
 		]);
 
 		$success = $response->getStatusCode() === 200;
+        $received = $success || $response->getStatusCode() === 406; // 406: Inactive recipient
```
Could you fix the indentation?

ruudvanderweijde replied:
ahhhh, this project is using tabs 😢

ruudvanderweijde replied:
Done

ruudvanderweijde committed:
Dont throw exception on 406

When the response code is 406, the api returned a valid response.

More info on error codes:
https://postmarkapp.com/developer/api/overview#error-codes
> 406 — Inactive recipient You tried to send email to a recipient
> that has been marked as inactive. Inactive recipients have either
> generated a hard bounce or a spam complaint. In this case, only
> hard bounce recipients can be reactivated by searching for them on
> your server’s Activity page and clicking the “Reactivate” button.
